src/symtable/SymbolTable.ts

```diff
@@ -55,7 +48,7 @@ export class SymbolTable {
             val = flag;
         }
 
-        if (curSte.comp_iter_target) {
+        if (curSte.compIterTarget) {
```

Comment:
I just make the yellow go away 😬

Reply:
🤣 
`deno-lint` policy
We could do something drastic and just use `camelCase` everywhere.
Which would mean adjusting our generated code.
It would be nice to get rid of `deno-lint-ignore camelcase`
And then the only snake case ends up being stuff that is python exposed. 
Which is basically nothing in this repo.
A potenial counter is that it moves us away from cpython. But I don't think it really does.

Reply:
yeah I don't know either, I went with camelCase for this because of the linter, and it didn't make much of a difference with cpython to be honest.